CHANGES.rst

```diff
@@ -25,6 +25,8 @@ Bugs fixed
 * #12162: Fix a performance regression in the C domain that has
   been present since version 3.0.0.
   Patch by Donald Hunter.
+* #12320: Fix removal of anchors from search summaries.
```

Comment:
Possibly also worth mentioning that this is a regression since 7.3.0 (comparing to the entry a couple of lines earlier).

Reply:
Yep, I'll add a note on that.